File: my-folder/2571-find-the-pivot-integer/solution.java
import java.lang.Math;
class Solution {
    public int pivotInteger(int n) {
        
        if(n == 1){ return n;}

        for(int i = 1; i < n; i++){
            int s1 = (i) * (i+1) / 2; 
            int s2 = (n - i + 1) * (i+n)/2;
            if (s1 == s2){
                return i;
            }
        }
        return -1; 
    }
}

File: my-folder/2571-find-the-pivot-integer/solution.py
class Solution(object):
    def pivotInteger(self, n):
        """
        :type n: int
        :rtype: int
        """
        if(n == 1): return 1
        l = 0
        r = n * (n + 1) // 2

        for i in range(1, n):
            l += i
            r -= (i - 1)
            
            if l == r:
                return i
        
        return -1
